src/screens/signup/signup2.tsx

```diff
@@ -25,8 +25,8 @@ export default function SignUpScreenTwo(props: SignupScreenTwoProp) {
   const { colors, fonts } = useThemeContext();
 
   const [email, setEmail] = useState('clairetamara08@gmail.com');
```

Comment:
and don't use this email state value as a place holder for the email input field rather use a placeholder

```javascript
  <Prompt>CREATE ACCOUNT</Prompt>
        <Input
          defaultValue={email} // use a text => 'clairetamara08@gmail.com'
          onChangeText={input => setEmail(input)}
          placeholder={email}
          textInputStyle={{
            height: 60,
            fontSize: fonts.LARGE_SIZE,
            marginBottom: '8%'
          }}
        />

```